docs/src/plasma/qtplasmac.adoc

```diff
@@ -1684,7 +1684,7 @@ The default LinuxCNC path tolerance is set for maximum speed which will severely
 
 It is recommended that the path tolerance is set by placing the appropriate G64 command and P value in the header of each G-code file.
 
-The provided G-code filter program will test for the existence of a G64 P__n__ command prior to the first motion command. If no G64 command is found it will insert a G64 P0.1 command which sets the path tolerance to 0.1 mm. For a imperial config the command will be G64 P0.004.
+The provided G-code filter program will test for the existence of a G64 P__n__ command prior to the first motion command. If no G64 command is found it will insert a G64 P0.1 command which sets the path tolerance to 0.1&thinsp;mm. For a imperial config the command will be G64 P0.004.
```

Comment:
```suggestion
The provided G-code filter program will test for the existence of a <<gcode:g64,`G64 P__n__`>> command prior to the first motion command. If no G64 command is found it will insert a G64 P0.1 command which sets the path tolerance to 0.1&thinsp;mm. For a imperial config the command will be G64 P0.004.
```